apps/api_web/lib/api_web/views/service_view.ex

```diff
@@ -7,6 +7,10 @@ defmodule ApiWeb.ServiceView do
     :start_date,
     :end_date,
     :valid_days,
+    :service_description,
+    :service_schedule_name,
+    :service_schedule_type,
+    :service_schedule_typicality,
```

Comment:
These are already part of the `service` endpoint, so the `service_` prefix is unnecessary.